pkg/apis/events/validation/validation.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/kubernetes/pkg/api"
+)
+
+// ValidateEvent validates a Event.
+func ValidateEvent(event *api.Event) field.ErrorList {
```

Comment:
no need for this file. The validation is only needed where the internal types are.

Reply:
Where this logic should live?

Reply:
I think you should just use the `pkg/api/validation/events.go`.

Reply:
How about this comment?

Reply:
Done.